### ISSUE TITLE
Conflicting with symfony/dotenv >= 5.1

### ISSUE THREAD

**Issue by infabo** [helhum/dotenv-connector#25]
I guess this is not enough:

https://github.com/helhum/dotenv-connector/commit/9ce9bb6ba94e09536b7345ad22a42bdcfbdfef90

What actually happens:

```
 - Updating symfony/dotenv (v5.0.9 => v5.1.2): Downloading (100%)         
  - Downgrading helhum/dotenv-connector (v2.3.1 => v2.3.0): Loading from cache
```

So composer just downgrades helhum/dotenv-connector cause of conflict-declaration in 2.3.1.

I suggest to use an apporpriate "require" like

```
"require": {
        "symfony/dotenv": "<5.1.0"
    }
```

**Comment by helhum** [comment]
> I guess this is not enough:
> What actually happens:
> Downgrading helhum/dotenv-connector (v2.3.1 => v2.3.0)

Yes, that is true. I cannot fix soemthing in already released versions.

> I suggest to use an apporpriate "require" like

That won't help either, as version 2.3.0 will still exist and will be resolved by Composer. You can fix this by requiring `^2.3.1` in your project.

I will fix sf/dotenv 5.1.0 compatibility

**Comment by infabo** [comment]
> You can fix this by requiring ^2.3.1 in your project.

Thanks. Didn't think of that!
